fern/pages/changelogs/docs/2024-11-27.mdx

```diff
@@ -0,0 +1,11 @@
+## Auth token autocompletion in API Explorer
+
+Auth tokens are now automatically injected into API explorer after authenticating to your API.
```

Comment:
`Save developers the hassle of finding and copying their API key. When authenticated, their API credentials will be automatically filled into the API Playground. This way, they can make their first API call even faster.` 

is the verbiage we used in our slack announcement